packages/arb-token-bridge-ui/src/util/L2NativeUtils.ts

```diff
@@ -8,6 +8,14 @@ export type L2NativeToken = {
   logoURI: string
 }
 
+export const ArbOneNativeUSDC = {
+  name: 'USD Coin',
+  symbol: 'USDC',
+  address: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
```

Comment:
I'd prefer re-using `CommonAddress.ArbitrumOne.USDC` here for maintainability instead of hardcoding the address again.